src/Context/RestContext.php

```diff
@@ -241,4 +241,17 @@ public function theRestHeaderShouldExistBe($header, $contents)
                 ' (actual: '.$header1[0].')');
         }
     }
+
+   /**
+   * @Then I should see :text in the JSON
+   */
+  public function iShouldSee($text)
```

Comment:
I think the method name should match the scenario description, i.e. both should say they're JSON-specific or neither should (and this should accurately match the implementation)